test/ref-list.json

```diff
@@ -121,7 +121,7 @@
     "ANNOTATION-MODEL-20151015",
     "ANNOTATION-PROTOCOL",
     "ANNOTATION-PROTOCOL-20150702",
-    "ANSI-X9-44-2007",
```

Comment:
Can you please add this one back in there an alias it to "ANSI-X9-44" like so:

```json
"ANSI-X-9-44-2007```: {
    "aliasOf": "ANSI-X-9-44"
}
```

We want to make sure never to break backwards comp. (See the bottom of the README for more on how to contribute.)